packages/paste-core/components/alert-dialog/stories/index.stories.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react';
+import {AlertDialog} from '../src';
+
+// Create FC's of all the stories you want to see/test
+// Should be 3: 2 choices regs, 2 choices destructive 1 choice notification
+
+export const AlertDialogExample: React.FC = () => {
+  return (
+    <AlertDialog
+      heading="Submit Application"
+      isOpen
+      primaryAction={() => window.alert()}
+      primaryLabel="Submit"
+      secondaryAction={() => {}}
+      secondaryLabel="Cancel"
+    >
+      Are you sure you want to submit this application? No information can be changed after submitting.
+    </AlertDialog>
+  );
+};
+
+export const AlertDialogForNotifying: React.FC = () => {
+  return (
+    <AlertDialog heading="Site Maintenance" isOpen primaryAction={() => {}} primaryLabel="Okay">
+      This site will be undergoing maintenance from 2:00 am 5/25/2020- 4:00am 5/26/2020. You will be unable to access
+      your account during this time.
+    </AlertDialog>
+  );
+};
+
+export const AlertDialogDestructive: React.FC = () => {
+  return (
+    <AlertDialog
+      heading="Delete Data"
+      isOpen
+      destructive
+      primaryAction={() => window.alert()}
+      primaryLabel="Delete"
+      secondaryAction={() => {}}
+      secondaryLabel="Cancel"
+    >
+      Are you sure you want to delete this data? This action cannot be undone.
+    </AlertDialog>
+  );
+};
+
+// export the stories but include 'excludeStories' of the FC's just made here
+// eslint-disable-next-line import/no-default-export
+export default {
+  title: 'Components/Alert-dialog',
+  // this is where excludeStories goes
+  excludeStories: ['AlertDialogExample', 'AlertDialogForNotifying', 'AlertDialogDestructive'],
+  component: AlertDialog,
+};
+
+// export the story ReactNodes by just returning the FC you want to use for it.
+// do these names use the proper convention?
+export const AlertDialogWithTwoActions = (): React.ReactNode => {
+  return <AlertDialogExample />;
+};
+
+export const AlertDialogWithOneAction = (): React.ReactNode => {
+  return <AlertDialogForNotifying />;
+};
+
+export const DestructiveAlertDialog = (): React.ReactNode => {
+  return <AlertDialogDestructive />;
+};
```

Comment:
Because you're not adding any metadata to the ReactNode examples here, you could just use the functional components as your stories and remove the `excludeStories` line. Storybook will use the FCs as the stories and you can export those ones for the tests as well!

Reply:
I remember us talking about this now! I think it'll mean I have to change the names of the FCs and therefore edit the tests, but this does seem like a more straightforward way to connect the two. Will do!